src/helpers/rerun-pr-workflows.ts

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2021 Expedia, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { context as githubContext } from '@actions/github';
+import { octokit } from '../octokit';
+
+export class RerunPrWorkflows {
+  branch = '';
+  //maybe fork name should be an optional input?
```

Comment:
i'm thinking that maybe if we keep the trigger on `pull_request` instead of `pull_request_target` it should automatically get the fork from the context. we likely couldn't force reruns on a fork from `pull_request_target` anyway as the tokens/PATs wouldn't have scope into the fork repo (i'm guessing)